model/client4.go

```diff
@@ -4353,6 +4353,9 @@ func (c *Client4) GetChannelsForScheme(schemeId string, page int, perPage int) (
 
 // Plugin Section
 
+const PLUGIN_URL_STRING = "plugin_download_url"
```

Comment:
OR

`download_url` and `signature_url`? that way the terminology will be the same across, server, redux, webapp

Reply:
@ali-farooq0 `plugin_download_url` has been released in 5.14 though. It's already documented [here](https://api.mattermost.com/#tag/plugins%2Fpaths%2F~1plugins~1install_from_url%2Fpost). This would break backward compatibility.

Reply:
Alright looks like i've been vetoed :P
Sticking with `plugin_download_url` and `signature_download_url`.